ubtue_SAGE Journals.js

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
This is a "fresh" source file for the SAGE translator. Is this really intended (cf. changes in SAGE\ Journals.js)

Reply:
as we just discussed I have deleted this file